bodhi/server/models.py

```diff
@@ -1075,9 +1080,11 @@ def get_pkg_committers_from_pagure(self):
         """
         pagure_url = config.get('pagure_url')
         # Pagure uses plural names for its namespaces such as "rpms" except for
-        # container
+        # container. Flatpaks build directly from the 'modules' namespace
```

Comment:
Ah, so I take this to mean that ACLs on a flatpak are identical to having ACLs to the module of the same name?